src/NuGet.Clients/NuGetConsole.Host.PowerShell/Utils/InstalledPackageEnumerator.cs

```diff
@@ -229,7 +230,7 @@ public override int GetHashCode()
 
             // Get the path to the Packages folder.
             var packagesFolderPath = packageManager.PackagesFolderSourceRepository.PackageSource.Source;
-            var packagePathResolver = new PackagePathResolver(packagesFolderPath);
+            var packagePathResolver = new PackagePathResolver(Path.GetFullPath(packagesFolderPath));
```

Comment:
@nkolev92 one could use `SourceUri` from the `PackageSource` here, but there's a risk that it might throw.